utils/constants.go

```diff
@@ -245,6 +245,29 @@ const (
 	Subscribe PubsubType = "subscribe"
 )
 
+const (
+	// GqlConnectionKeepAlive send every 20 second to client over websocket
+	GqlConnectionKeepAlive = "ka" // Server -> Client
```

Comment:
All these are not matching the types specified in the protocol. Everything should be in capital.